plugins/modules/lvg.py

```diff
@@ -156,6 +208,176 @@ def parse_pvs(module, data):
     return pvs
 
 
+def find_vg(module, vg):
+    vgs_cmd = module.get_bin_path('vgs', True)
+    rc, current_vgs, err = module.run_command("%s --noheadings -o vg_name,pv_count,lv_count --separator ';'" % vgs_cmd)
+
+    if rc != 0:
+        module.fail_json(msg="Failed executing vgs command.", rc=rc, err=err)
```

Comment:
Why not use `run_command()`'s `check_rc=True`?

Reply:
It makes sense.